docs/book/features/volume_expansion.md

```diff
@@ -1,120 +1,78 @@
 # vSphere CSI Driver - Offline Volume Expansion
 
-CSI Volume Expansion was introduced as an alpha feature in Kubernetes 1.14 and it was promoted to beta in Kubernetes 1.16. The Vanilla vSphere CSI driver supports offline volume expansion which allows a volume to be extended when it is not attached to a node. Currently the support is available only for block volumes.
```

Comment:
Does it make sense to call out that we don't support volume expansion in vSphere with Kubernetes cluster here? Is this called out somewhere else?

Reply:
Done